src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -676,6 +676,8 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
             'Populate the target framework combobox
             PopulateTargetFrameworkComboBox(Me.TargetFramework)
+
+            Me.AssemblyInfoButton.Visible = Not Utils.IsTargetingDotNetCore(Me.ProjectHierarchy)
```

Comment:
Hardcoding TFMs is definitely not the right approach. We're also enabling this for *desktop* projects that are based in CPS.